### PR TITLE
Oddities

### PR DESCRIPTION
**Pull request by SpadesSlick**
## About The Pull Request
<details>
<summary>
Underail like oddities that can permanently increase your stats.
</summary>
<hr>

## Changelog
:cl:
add: Oddities for stats level ups.

/:cl:

### CODE REVIEW

File: code/game/objects/items/oddities.dm [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+	icon_state = "old_newspaper"
+	oddity_stats = list(
+		STAT_MEC = 4,
+		STAT_COG = 4,
+		STAT_BIO = 4,
+	)
+
+/obj/item/weapon/oddity/common/paper_crumpled
+	name = "turn-out page"
+	desc = "This is ALMOST makes sense."
```
Comment (GrayRachnid) [inline code/game/objects/items/oddities.dm]: remove the is

File: code/game/objects/items/oddities.dm [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+	icon_state = "book_eyes"
+	oddity_stats = list(
+		STAR_ROB = 9,
+		STAT_THG = 9,
+		STAT_VIG = 9,
+	)
+
+/obj/item/weapon/oddity/common/book_omega
+	name = "occult book"
+	desc = "That's mostly does not make any logical sense. However, those stories are at least interesting."
```
Comment (GrayRachnid) [inline code/game/objects/items/oddities.dm]: 'This mostly does not make any sense.'

Comment (SpadesSlick) [reply code/game/objects/items/oddities.dm]: Fixed.

File: code/game/objects/items/oddities.dm (hunk 1 above)
Comment (SpadesSlick) [reply code/game/objects/items/oddities.dm]: Fixed.

**Comment by SpadesSlick** [comment]
Ready for review.

File: code/game/objects/items/oddities.dm [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+
+
+//Oddities are separated into categories depending on their origin. They are meant to be used both in maints and derelicts, so this is important
+//This is done by subtypes, because this way even densiest code monkey will not able to misuse them
+//They are meant to be put in appropriate random spawners
+
+//Common - you can find those everywhere
+/obj/item/weapon/oddity/common/blueprint
+	name = "strange blueprint"
+	desc = "You can't figure out what exactly this is. This is probably don't even works."
```
Comment (granodd) [inline code/game/objects/items/oddities.dm]: Should be written as: "There's no telling what this design is supposed to be. Whatever could be built from this likely wouldn't work."

File: code/game/objects/items/oddities.dm [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+	icon_state = "blueprint"
+	oddity_stats = list(
+		STAT_COG = 5,
+		STAT_MEC = 7,
+	)
+
+/obj/item/weapon/oddity/common/coin
+	name = "strange coin"
+	desc = "It look more like collectible rather than actual curency. But you can't figure out the alloy it made from."
```
Comment (granodd) [inline code/game/objects/items/oddities.dm]: "It appears to be more of a collectible than any sort of actual currency. What metal it's made from seems to be a mystery."

File: code/game/objects/items/oddities.dm [hunk 5]
```diff
@@ -0,0 +1,211 @@
...
+	icon_state = "coin"
+	oddity_stats = list(
+		STAT_ROB = 5,
+		STAT_THG = 5,
+	)
+
+/obj/item/weapon/oddity/common/photo_landscape
+	name = "alien landscape photo"
+	desc = "There some ire about this planet."
```
Comment (granodd) [inline code/game/objects/items/oddities.dm]: "There is some ire about the planet in this photograph."

File: code/game/objects/items/oddities.dm [hunk 6]
```diff
@@ -0,0 +1,211 @@
...
+	icon_state = "photo_landscape"
+	oddity_stats = list(
+		STAT_COG = 5,
+		STAT_THG = 5,
+	)
+
+/obj/item/weapon/oddity/common/photo_coridor
+	name = "surreal maint photo"
+	desc = "Something wrong, yet so familiar about this place."
```
Comment (granodd) [inline code/game/objects/items/oddities.dm]: "This corridor looks familiar, though something seems wrong about it, as if everything in it was replaced with an exact replica of itself."

File: code/game/objects/items/oddities.dm [hunk 7]
```diff
@@ -0,0 +1,211 @@
...
+	icon_state = "photo_corridor"
+	oddity_stats = list(
+		STAT_MEC = 5,
+		STAT_THG = 5,
+	)
+
+/obj/item/weapon/oddity/common/photo_eyes
+	name = "observer photo"
+	desc = "What the fuck is this even."
```
Comment (granodd) [inline code/game/objects/items/oddities.dm]: "Just looking at this photo sparks a primal fear in your heart."

File: code/game/objects/items/oddities.dm (hunk 2 above)
Comment (granodd) [inline code/game/objects/items/oddities.dm]: "Most of the stories in this book seem to be the writing of mad men. But at least the stories are interesting."

File: code/game/objects/items/oddities.dm [hunk 8]
```diff
@@ -0,0 +1,211 @@
...
+	desc = "Oh, how quickly we forgot."
+	icon_state = "book_bible"
+	oddity_stats = list(
+		STAR_ROB = 5,
+		STAT_VIG = 5,
+	)
+
+/obj/item/weapon/oddity/common/old_money
+	name = "old_money"
+	desc = "It's not like organization that issues this exist now."
```
Comment (granodd) [inline code/game/objects/items/oddities.dm]: "Legal tender to a government that likely no longer exists."

File: code/game/objects/items/oddities.dm [hunk 9]
```diff
@@ -0,0 +1,211 @@
...
+	icon_state = "healthscanner"
+	item_state = "electronic"
+	oddity_stats = list(
+		STAT_COG = 8,
+		STAT_BIO = 8,
+	)
+
+/obj/item/weapon/oddity/common/old_pda
+	name = "broken pda"
+	desc = "This is old, Nanotrasen era PDA, issued to employes of this corporation all over the galaxy."
```
Comment (granodd) [inline code/game/objects/items/oddities.dm]: "An old Nanotrasen era PDA. These were issued to their employees all throughout the galaxy."

File: code/game/objects/items/oddities.dm [hunk 10]
```diff
@@ -0,0 +1,268 @@
...
+	desc = "There's no telling what this design is supposed to be. Whatever could be built from this likely wouldn't work."
+	icon_state = "blueprint"
+	oddity_stats = list(
+		STAT_COG = 5,
+		STAT_MEC = 7,
+	)
+
+/obj/item/weapon/oddity/common/coin
+	name = "strange coin"
+	desc = "It appears to be more of a collectible than any sort of actual currency. What metal it's made from seems to be a mystery.."
```
Comment (granodd) [inline code/game/objects/items/oddities.dm]: missing a period in the ellipses at the end

File: code/game/objects/items/oddities.dm [hunk 11]
```diff
@@ -0,0 +1,268 @@
...
+	icon_state = "coin"
+	oddity_stats = list(
+		STAT_ROB = 5,
+		STAT_TGH = 5,
+	)
+
+/obj/item/weapon/oddity/common/photo_landscape
+	name = "alien landscape photo"
+	desc = "There is some ire about the planet in this photograph.."
```
Comment (granodd) [inline code/game/objects/items/oddities.dm]: also missing a third period in the ellipses

File: code/game/objects/items/oddities.dm [hunk 12]
```diff
@@ -0,0 +1,268 @@
...
+	icon_state = "photo_landscape"
+	oddity_stats = list(
+		STAT_COG = 5,
+		STAT_TGH = 5,
+	)
+
+/obj/item/weapon/oddity/common/photo_coridor
+	name = "surreal maint photo"
+	desc = "he corridor in this photograph looks familiar, though something seems wrong about it; as if everything in it was replaced with an exact replica of itself."
```
Comment (granodd) [inline code/game/objects/items/oddities.dm]: Missing the T at the start

**Pull request merged**